yarn/datadog_checks/yarn/yarn.py

```diff
@@ -340,7 +341,7 @@ def _yarn_node_metrics(self, rm_address, addl_tags):
                 tags.extend(addl_tags)
 
                 self._set_yarn_metrics_from_json(tags, node_json, YARN_NODE_METRICS)
-                version = node_json.get('version')
+                version = node_json.get('version', node_json.get('hadoopVersion'))
```

Comment:
In newer versions of yarn, they changed the metric being returned from `version` to `hadoopVersion`.  See https://hadoop.apache.org/docs/stable/hadoop-yarn/hadoop-yarn-site/NodeManagerRest.html